src/index.html

```diff
@@ -7,8 +7,53 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700&display=swap" rel="stylesheet">  
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="head">
+      <img 
+           src="images/logo.png" 
+           alt="logo" 
+           class="logo"
+      >
```

Comment:
fix indentation everywhere

```suggestion
      <img
        src="images/logo.png"
        alt="logo"
        class="logo"
      >
```